src/gf-anaconda-cleanup

```diff
@@ -30,11 +30,23 @@ EOF
     exit 1
 fi
 
-# Remove some things written by anaconda; eventually
-# we want to reset /etc except we also need /etc/fstab right now e.g.
-for x in "/etc/sysconfig/anaconda" "/etc/resolv.conf" "/etc/systemd/system/default.target"; do
-    coreos_gf rm-rf "${deploydir}${x}"
+# Remove most things written by anaconda in /etc, except
+# for /etc/fstab mainly.  We also keep /etc/hostname and /etc/locale.conf as otherwise
+# systemd-firstboot triggers.
+for x in "sysconfig/anaconda" "sysconfig/network" "crypttab" "resolv.conf" "X11" "systemd/system/default.target" "shadow-"; do
+    coreos_gf rm-rf "${deploydir}/etc/${x}"
 done
+for x in $(coreos_gf glob-expand "${deploydir}/etc/sysconfig/network-scripts/"'*'); do
+    coreos_gf rm-rf "${x}"
```

Comment:
Could probably just unite those two loops using `coreos_gf glob rm-rf` instead, but eh... not worth the respin.